bundle.Dockerfile

```diff
@@ -4,9 +4,9 @@ FROM scratch
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
```

Comment:
I think the changes in this file can be removed? I dont see this file getting updated for previous releases.